muvsfunc.py

```diff
@@ -5666,7 +5690,17 @@ def exclusive_accumulate(iterable: Iterable[T], func: Callable[[T, T], T] = oper
             ofile.writelines(olines)
 
     # Output spliced clip
-    return core.std.Splice(clips, mismatch=True)
+    output = core.std.Splice(clips, mismatch=True)
+
+    # Use a reference clip for CFR output.
+    if cfr_output:
+        clipa = output
+        clipb = ref
+        fpn_out = clipa.num_frames*clipb.fps.numerator/clipb.num_frames
+        fpd_out = clipb.fps.denominator
```

Comment:
Following the previous change,
```python3
fpd_out = clipb.fps.denominator * clipb.num_frames
```
such that no floating-point rounding error presented.

Reply:
Thanks!
I did that, but a trouble happend. In some situations, it will output with very big numbers, like `FPS: 17649330000/299009711 = 59.0259`, with codes:
```python
clip1 = core.std.BlankClip(format=vs.YUV420P10, length=298711, fpsnum=30000, fpsden=1001)
clip2 = core.std.BlankClip(format=vs.YUV420P10, length=597422, fpsnum=60000, fpsden=1001)

st_num = 289600
p0 = clip2[:st_num*2]
p1 = clip1[st_num:]
out = VFRSplice([p0, p1], tcfile="E:\\A\\Test\\test.txt", cfr_ref=clip2)
out.set_output()
```
And x265 will output a video with an incorrect frame rate info: 
![](https://i.slow.pics/dKCsAwcK.png)
although timecode file can correct it.

Reply:
I have no idea to fix it. ![](https://i.slow.pics/3WdBWwUw.gif)

Reply:
Oh thanks, I am not aware of that.